### DISCUSSION TITLE
Set default camera angle to any value (0-30)

### DISCUSSION THREAD

**Discussion by Tenome** [emoose/OutRun2006Tweaks#236] [Suggestions]
### Describe the feature you'd like to see added

I've used cheat engine to lock `0x0079FE10+0x34A` to 3 before starting a race, and it switches camera angles with no issue (this also lets you use the locked camera angles without having to set `0x484BAD` to 30, and will let you swap through the other unused angles, but once you reach camera angle 0 it'll reset back to the default angles). 

Can that be made into a Tweaks feature? Something like CameraAngle = 0-30

**Comment by emoose** [comment]
There is already a game setting that lets you change the default camera to use (think that setting only appears if you open settings from main menu though), maybe that could be extended somehow (again a C2C thing though so would probably take some work..)

**Comment by Tenome** [comment]
Oh yeah, this suggestion was made in mind with the possibility of using the expanded camera angles (and some new ones maybe). Forgot to mention that, my bad.
